app/src/main/java/com/myperishableplanner/v21001/dto/Item.kt

```diff
@@ -5,13 +5,13 @@ import com.google.gson.annotations.SerializedName
 data class Item(@SerializedName("fdcId") var id: Int, @SerializedName("description") var name:String, @SerializedName("brandOwner") var brand:String){
     override fun toString(): String {
 
-        var prefix : String = ""
```

Comment:
Can declare prefix as lateinit instead as it is more intuitive and fitting for the initializing the string later.